staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -660,6 +710,47 @@ See https://kubernetes.io/docs/reference/using-api/server-side-apply/#conflicts`
 	return nil
 }
 
+func (o *ApplyOptions) calculateSSAMigrationPatch(obj runtime.Object) ([]byte, error) {
+	upgraded := obj.DeepCopyObject()
+	if err := csaupgrade.UpgradeManagedFields(
+		upgraded,
+		"kubectl-client-side-apply",
```

Comment:
This hard-code is really unfortunate, on the other hand, I don't want people to specify two manager when they run kubectl...

Reply:
Unfortunate, but not a blocker. Is this enumerated as a constant somewhere?

Reply:
yes, there was `FieldManagerClientSideApply` in this very file. Will replace